commit 74d3abf46b42cf3b5f692c5276179c362c08d042
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-07-11T15:52:51-07:00 (3 months ago)
    
    TRITON-1777 sdc-migrate finalize does not output anything on success

diff --git a/bin/sdc-migrate b/bin/sdc-migrate
index ca5870f..27cec03 100755
--- a/bin/sdc-migrate
+++ b/bin/sdc-migrate
@@ -547,7 +547,9 @@ EOF
 
 
 function migration_finalize() {
+    local message
     local result
+    local status_code
     local url
     local vm_uuid=$1
 
@@ -569,12 +571,17 @@ EOF
 
     url="/vms/${vm_uuid}?action=migrate&migration_action=${action}"
 
-    result=$(sdc-vmapi "$url" -X POST | $JSON -q -H)
-    if [[ -z $result ]]; then
-        result="Done - the migration is finished."
+    result=$(sdc-vmapi "$url" -X POST)
+
+    # A 200 status code means the finalize was successful.
+    status_code=$(echo "$result" | grep '^HTTP' | head -1 | awk '{print $2}')
+    if [[ $status_code == "200" ]]; then
+        message="Done - the migration is finished."
+    else
+        message=$(echo "$result" | $JSON -q -H)
     fi
 
-    echo "$result"
+    echo "$message"
 }
 
 
diff --git a/package.json b/package.json
index 13a1a8f..3c1acf9 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.8.2",
+    "version": "1.8.3",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
